blocks/test/full-content.js

```diff
@@ -95,13 +95,23 @@ describe( 'full post content fixture', () => {
 
 	fileBasenames.forEach( f => {
 		it( f, () => {
+			/**
```

Comment:
We have three styles of multi-line comments:

- [WordPress standard](https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#comments) `/*`
- [Calypso standard](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#comments) `//`
- This `/**`